webapp/.project

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<projectDescription>
+	<name>webapp</name>
+	<comment></comment>
+	<projects>
+	</projects>
+	<buildSpec>
+	</buildSpec>
+	<natures>
+	</natures>
+</projectDescription>
```

Comment:
I think this may be a file specific to using Visual Studio: This shouldn't be in the repo. I added it to .gitignore

Reply:
Alright, thank you didn't even notice.